### PR TITLE
Custom style in git-clang-format

### PR DESCRIPTION
**Pull request by dogoepp**
With these modifications, we can use the same clang-format style everywhere, automatically.

@costashatz @jbmouret @vassilisvas  please test on your computer, so that we can start using this tool.

### CODE REVIEW

**Comment by dogoepp** [comment]
Once accepted, we shall merge the related [pull request of limbo](https://github.com/resibots/limbo/pull/143).

**Comment by costashatz** [comment]
~~Seems good to me. I cannot really review it though..~~

File: git-clang-format [hunk 1]
```diff
@@ -144,10 +148,22 @@ def main():
   # The computed diff outputs absolute paths, so we must cd before accessing
   # those files.
   cd_to_toplevel()
+
+  # If told to, temporarily retrieve a .clang-format style file
+  to_delete = False
+  if opts.style_path:
+    to_delete = copy_format_file(opts.style_path)
+    if opts.verbose >= 1:
+      if already_exists:
```
Comment (costashatz) [inline git-clang-format]: When running the script, I get `NameError: global name 'already_exists' is not defined`..

Comment (dogoepp) [inline git-clang-format]: Thanks a lot, fixed.

**Comment by costashatz** [comment]
Now it's good.. Nice tool btw.. Thanks..! :smile: 

Let's **merge**..

**Comment by dogoepp** [comment]
Thanks !

**Merging**

**Pull request merged**